SCClassLibrary/Common/Control/Score.sc

```diff
@@ -3,6 +3,7 @@ Score {
 	classvar <>program, <>options;
 
 	*initClass {
+		Class.initClassTree(ServerOptions);
```

Comment:
leaving a comment here would be nice - something like 'in order to have default ServerOptions values available'

Reply:
done